resources/validations/src/CIS.xsl

```diff
@@ -0,0 +1,463 @@
+                            <xsl:value-of
+                                select="$catalog//metadata/title" />
+                        </cite>
+                    </a>
+                    <xsl:text>: version </xsl:text>
+                    <code
+                        class="version">
+                        <xsl:value-of
+                            select="$catalog//metadata/version" />
+                    </code>
+                    <xsl:text> last modified </xsl:text>
+                    <xsl:value-of
+                        select="format-dateTime($catalog//metadata/last-modified, '[MNn] [D] [Y]')" />
+                    <xsl:text>.</xsl:text>
+                </p>
+                <p>There are <xsl:value-of
+                        select="count($catalog//control)" /> required controls at that security-sensitivity-level having a total of <xsl:value-of
+                        select="
+                            count($catalog//prop[@name = 'response-point']/parent::part[matches(@id, 'smt')])" /> response
+                    points.</p>
+
+                <xsl:call-template
+                    name="nested" />
+
+            </body>
+
+        </html>
+    </xsl:template>
+
+    <xsl:template
+        name="nested">
+        <h2>Nested Control Implementation Summary</h2>
+        <details>
+            <summary>Legend</summary>
+            <table>
+                <caption>Symbol Usage</caption>
+                <thead>
+                    <tr>
+                        <th
+                            class="c">Symbol</th>
+                        <th>Denotation</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td
+                            class="c">⦿</td>
+                        <td>A FedRAMP Core (<a
+                                target="_blank"
+                                href="https://www.fedramp.gov/assets/resources/documents/CSP_Annual_Assessment_Guidance.pdf"><cite>FedRAMP Annual
+                                    Assessment Guidance</cite></a> §2.3.1) control</td>
+                    </tr>
+                    <tr>
+                        <td
+                            class="c implemented">✓</td>
+                        <td>A control has an implementation</td>
+                    </tr>
+                    <tr>
+                        <td
+                            class="c partially-implemented">◐</td>
+                        <td>A control has a partial implementation</td>
+                    </tr>
+                    <tr>
+                        <td
+                            class="c planned">⏳</td>
+                        <td>A control has a planned implementation</td>
+                    </tr>
+                    <tr>
+                        <td
+                            class="c not-implemented">⁇</td>
+                        <td>A control lacks an implementation</td>
```

Comment:
The answer, such as it is, were to present a set of rows each corresponding to a control in the worksheet for each related statement (response point) as **entirely empty** except for the cells in column A. An example of an unimplemented control would be OSCAL statement IDs - which are also specific response points -  ac-1_smt.a.1, ac-1_smt.a.2, ac-1_smt.b.1, ac-1_smt.b.2 which somewhat correspond to the rather sloppy identifiers in the worksheet "AC-01 (a)", "AC-01 (b)". One would likely have to stretch the correspondence to have "AC-01 (a)" address the two related response points.

Reply:
The current validations which have been brought forward from phase 2 have [several assertions](https://github.com/18F/fedramp-automation/blob/develop/resources/validations/src/ssp.sch#L262-L276) which at the document (as opposed to individual control) level demand responses. These are augmented by several [context-specific rules+assertions](https://github.com/18F/fedramp-automation/blob/develop/resources/validations/src/ssp.sch#L286-L394) which elaborate the relevant assertions. The manner in which successful or failed Schematron assertions might be presented is not quite the same as the worksheets since the worksheet fails to identify all individual mandatory response points.

Reply:
If the UI ends up being the **sole** origin (a prospect which seems to me a bit odd) of submitted OSCAL SSP validation information, augmented by other quality estimations, as well as additionally resembling an analog of a multi-hundred page document meticulously crafted using a text manipulation application considered state of the art a half century ago when document renditions suitable for printing - on paper - were considered the apical expression of good form and taste, then there will be a large amount of UI work. I would expect that the UI would minimally have to provide a credible presentation of an SSP as rendered on a clay tablet, or adopt an alternative form suitable for use on a computer workstation equipped with a contemporary web browser.

Reply:
The CIS worksheets do not cover all response points. Regardless, inadequately defined control expressions would have rows with blank columns. That rendition, in my opinion, is Not Good.

In addition to [document-level assertions](https://github.com/18F/fedramp-automation/blob/develop/resources/validations/src/ssp.sch#L233-L285), there are [per-control ones](https://github.com/18F/fedramp-automation/blob/develop/resources/validations/src/ssp.sch#L286-L394). These do not resemble the CIS worksheets.

We could create a rendition like the CIS worksheets. The FedRAMP FIPS 199 High catalog has 421 required controls having a total of 746 response points.